src/boards.c

```diff
@@ -751,10 +821,15 @@ void st7789_state(int state)
       if (bp >= sizeof(linebuffer)) {
 	/* currently we have a single image which we clip to show different modes */
 	if (topclip) {
-	  if (y < 48)
+	  if (y < 48) {
 	    memset(linebuffer, 0, sizeof(linebuffer));
-	  else
+	  } else if (y < 76) {
+	    #ifdef _P8_NRF52832_H
+	    memset(linebuffer+2*(240-54), 0, 54*2);
+	    #else
```

Comment:
I presume this is mirroring to put the Start -> arrow on the other side. Why does the clip position change as well?

Also can we make this #ifdef BUTTON_ON_RIGHT and set that in p8_nrf52832/board.h (effectively making the code self commenting).

Reply:
Yes, the Start -> arrow points to the right and clip position changes to remove the arrow as it's now on the right hand side of the display.
If y < 48, clip the whole line to remove the word start (same for PT and P8).
Then the (original) Pinetime clip blanks the first 64 display bits all the way to the bottom to remove the arrow.
The P8 clip blanks the last 54 display bits to remove the arrow when y < 76.  If the code clips all the way down to the bottom (like the PT code) it blanks the BT icon which is in the bottom right corner.

I have updated with #ifdef BUTTON_ON_RIGHT